translations/translate.js

```diff
@@ -1447,5 +1447,441 @@ translate = {
""},{"n":"Thai","a2":"th","a3b":"tha","a3t":""},{"n":"Tibetan","a2":"bo","a3b":"tib","a3t":"bod"},{"n":"Tigre","a2":"","a3b":"tig","a3t":""},{"n":"Tigrinya","a2":"ti","a3b":"tir","a3t":""},{"n":"Tiv","a2":"","a3b":"tiv","a3t":""},{"n":"Tokelau","a2":"","a3b":"tkl","a3t":""},{"n":"Klingon; tlhIngan-Hol","a2":"","a3b":"tlh","a3t":""},{"n":"Tlingit","a2":"","a3b":"tli","a3t":""},{"n":"Tamashek","a2":"","a3b":"tmh","a3t":""},{"n":"Tonga (Nyasa)","a2":"","a3b":"tog","a3t":""},{"n":"Tonga (Tonga Islands)","a2":"to","a3b":"ton","a3t":""},{"n":"Tok Pisin","a2":"","a3b":"tpi","a3t":""},{"n":"Tsimshian","a2":"","a3b":"tsi","a3t":""},{"n":"Tswana","a2":"tn","a3b":"tsn","a3t":""},{"n":"Tsonga","a2":"ts","a3b":"tso","a3t":""},{"n":"Turkmen","a2":"tk","a3b":"tuk","a3t":""},{"n":"Tumbuka","a2":"","a3b":"tum","a3t":""},{"n":"Tupi languages","a2":"","a3b":"tup","a3t":""},{"n":"Turkish","a2":"tr","a3b":"tur","a3t":""},{"n":"Altaic languages","a2":"","a3b":"tut","a3t":""},{"n":"Tuvalu","a2":"","a3b":"tvl","a3t":""},{"n":"Twi","a2":"tw","a3b":"twi","a3t":""},{"n":"Tuvinian","a2":"","a3b":"tyv","a3t":""},{"n":"Udmurt","a2":"","a3b":"udm","a3t":""},{"n":"Ugaritic","a2":"","a3b":"uga","a3t":""},{"n":"Uighur; Uyghur","a2":"ug","a3b":"uig","a3t":""},{"n":"Ukrainian","a2":"uk","a3b":"ukr","a3t":""},{"n":"Umbundu","a2":"","a3b":"umb","a3t":""},{"n":"Undetermined","a2":"","a3b":"und","a3t":""},{"n":"Urdu","a2":"ur","a3b":"urd","a3t":""},{"n":"Uzbek","a2":"uz","a3b":"uzb","a3t":""},{"n":"Vai","a2":"","a3b":"vai","a3t":""},{"n":"Venda","a2":"ve","a3b":"ven","a3t":""},{"n":"Vietnamese","a2":"vi","a3b":"vie","a3t":""},{"n":"Volapük","a2":"vo","a3b":"vol","a3t":""},{"n":"Votic","a2":"","a3b":"vot","a3t":""},{"n":"Wakashan languages","a2":"","a3b":"wak","a3t":""},{"n":"Wolaitta; Wolaytta","a2":"","a3b":"wal","a3t":""},{"n":"Waray","a2":"","a3b":"war","a3t":""},{"n":"Washo","a2":"","a3b":"was","a3t":""},{"n":"Welsh","a2":"cy","a3b":"wel","a3t":"cym"},{"n":"Sorbian languages","a2":"","a3b":"wen","a3t":""},{"n":"Walloon","a2":"wa","a3b":"wln","a3t":""},{"n":"Wolof","a2":"wo","a3b":"wol","a3t":""},{"n":"Kalmyk; Oirat","a2":"","a3b":"xal","a3t":""},{"n":"Xhosa","a2":"xh","a3b":"xho","a3t":""},{"n":"Yao","a2":"","a3b":"yao","a3t":""},{"n":"Yapese","a2":"","a3b":"yap","a3t":""},{"n":"Yiddish","a2":"yi","a3b":"yid","a3t":""},{"n":"Yoruba","a2":"yo","a3b":"yor","a3t":""},{"n":"Yupik languages","a2":"","a3b":"ypk","a3t":""},{"n":"Zapotec","a2":"","a3b":"zap","a3t":""},{"n":"Blissymbols; Blissymbolics; Bliss","a2":"","a3b":"zbl","a3t":""},{"n":"Zenaga","a2":"","a3b":"zen","a3t":""},{"n":"Standard Moroccan Tamazight","a2":"","a3b":"zgh","a3t":""},{"n":"Zhuang; Chuang","a2":"za","a3b":"zha","a3t":""},{"n":"Zande languages","a2":"","a3b":"znd","a3t":""},{"n":"Zulu","a2":"zu","a3b":"zul","a3t":""},{"n":"Zuni","a2":"","a3b":"zun","a3t":""},{"n":"No linguistic content; Not applicable","a2":"","a3b":"zxx","a3t":""},{"n":"Zaza; Dimili; Dimli; Kirdki; Kirmanjki; Zazaki","a2":"","a3b":"zza","a3t":""}
```

Comment:
I think this line goes over the 80 column limit!!!  Since it is a static list, it is perfectly fine to have it this way.